src/Core/Vipr/Bootstrapper.cs

```diff
@@ -30,15 +30,17 @@ internal class Bootstrapper
         private string _readerName = "ODataReader.v4";
         private string _writerName = "CSharpWriter";
         private string _metadataPath = "http://services.odata.org/V4/TripPinServiceRW/$metadata";
-        private string _outputPath = "";
```

Comment:
empty string or dot - e.g. `--outputPath=""` or `--outputPath="."` or `--outputPath=".\"`
I added a test for the empty string case.